### PR TITLE
Added default_content module

### PR DESCRIPTION
**Pull request by jrosgiralt**

### CODE REVIEW

File: composer.lock [hunk 1]
```diff
@@ -2124,6 +2124,83 @@
             },
```
Comment (jrosgiralt) [inline composer.lock:2124]: @greggmarshall could you verify that this modification looks of to you? I had a merge conflict with the hashes when switching to my branch which required stashing, and I had to manually pick one of the two hash value options.

**Comment by greggmarshall** [comment]
Easiest solution when you get a merge conflict with the hashes, is pick one.  Then after the merge (or during it before committing the composer.lock file) do a
````composer update --lock````
to get the correct hash.  Commit that.

**Comment by jrosgiralt** [comment]
OK, I run ```composer update --lock```. It asked me to install phpstan/extension-installer, which I did. See the updated commit. Thks.

**Comment by jrosgiralt** [comment]
@greggmarshall is this good to be merged? many thanks

**Comment by jrosgiralt** [comment]
@greggmarshall is this good to be merged? many thanks

**Review by greggmarshall — APPROVED** [review]

**Comment by jrosgiralt** [comment]
It looks like this merge broke the main branch, which now does not build correctly:

```
// Import the listed configuration changes?: yes.                                                                      

 [error]  Drupal\Core\Config\ConfigImporterException: There were errors validating the config synchronization.
Unable to install the <em class="placeholder">default_content_test</em> module since it does not exist. in Drupal\Core\Config\ConfigImporter->validate() (line 756 of /var/www/html/web/core/lib/Drupal/Core/Config/ConfigImporter.php). 

In ConfigImportCommands.php line 318:
                                                                                                           
  The import failed due to the following reasons:                                                          
  Unable to install the <em class="placeholder">default_content_test</em> module since it does not exist.  
                                                                                                           
```

image

I need to leave now and won't be able to look at this right away, but will try to pick it up soon.

**Comment by jrosgiralt** [comment]
I see what the error is, it's in a module I created but did not commit, and the flag to install it is on. I will be fixing this soon.

**Comment by jrosgiralt** [comment]
This is being fixed in this other PR: https://github.com/Labdoo/Labdoo-3.0/pull/26

**Pull request merged**